src/main/java/net/glowstone/util/OpenCompute.java

```diff
@@ -92,32 +122,21 @@ public static void initContext(CLPlatform platform) {
         GlowServer.logger.info("OpenCL: Using " + platform + " on device " + device + ".");
     }
 
-    public static CLPlatform getPlatform() {
-        return platform;
-    }
-
-    public static CLContext getContext() {
-        return context;
-    }
-
-    public static CLDevice getDevice() {
-        return device;
-    }
-
-    public static CLCommandQueue getQueue() {
-        return queue;
-    }
-
+    /**
+     * Calculates the number of work groups.
+     * @param size the total number of local work units
+     * @return the number of work groups
+     */
     public static int getGlobalSize(int size) {
-        int globalSize = size;
-        int localWorkSize = getLocalSize();
-        int r = globalSize % localWorkSize;
-        if (r != 0) {
-            globalSize += localWorkSize - r;
-        }
-        return globalSize;
+        return getGlobalSize(size, getLocalSize());
     }
 
+    /**
+     * Calculates the number of work groups.
```

Comment:
New line between description and params.

Reply:
Done.

Reply:
@Pr0methean please do the same thing with other Javadocs in this PR.

Reply:
Done in 283d634